cmd/zed/api/command.go

```diff
@@ -52,17 +54,34 @@ func DefaultHost() string {
 func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	c := &Command{Command: parent.(*root.Command)}
 	f.StringVar(&c.Host, "host", DefaultHost(), "host[:port] of Zed lake service")
+	f.StringVar(&c.configPath, "configpath", "", "directory to store local configuration and credentials")
```

Comment:
I think we should set the default here rather than using zapiPath.
```suggestion
	dir, _ = os.UserHomeDir()
	if dir != "" {
		dir := filepath.Join(dir, "zapi")
	}
	c := &Command{Command: parent.(*root.Command)}
	f.StringVar(&c.Host, "host", DefaultHost(), "host[:port] of Zed lake service")
	f.StringVar(&c.configDir, "configDir", dir, "configuration and credentials directory")
```

Reply:
Why not